OpenRA.Mods.Common/Effects/FlashTarget.cs

```diff
@@ -21,12 +21,14 @@ public class FlashTarget : IEffect
 		Actor target;
 		Player player;
 		int remainingTicks;
+		int delay;
 
-		public FlashTarget(Actor target, Player asPlayer = null, int ticks = 4)
+		public FlashTarget(Actor target, Player asPlayer = null, int ticks = 4, int delay = 0)
 		{
 			this.target = target;
 			player = asPlayer;
 			remainingTicks = ticks;
+			this.delay = delay;
 			target.World.RemoveAll(effect =>
```

Comment:
The regression with only one flash showing still exists, and happens because of this block.  This is an important work-around for when multiple different things flash the same target, so removing it isn't a good option.  I think a better fix would be to have `count` and `stride` parameters instead of `delay`, and make the one `FlashTarget` responsible for all the flashes in the sequence.